test/functional/specs/Personalization/C6364800.js

```diff
@@ -90,6 +90,11 @@ const getAepEdgeResponse = async requestId => {
           {
             key: "kndctr_97D1F3F459CE0AD80A495CBE_AdobeOrg_cluster",
             value: "or2"
+          },
+          {
+            key: "at_qa_mode",
+            value:
+              '{"token":"xFgbBknmbC1SwBodlfVLxgF5KVrsCNsY4g8dvHk0wA0","listedActivitiesOnly":true,"previewIndexes":[{"activityIndex":1,"experienceIndex":1}]}'
           }
```

Comment:
Can you add a comment explaining why we need to have the qa_mode cookie for the test?